TOPICS.txt

```diff
@@ -63,11 +63,18 @@ Parsing
 Pattern recognition
 Performance
 Randomness
-Reactive programming
 Refactoring
 Regular Expressions
 Searching
 Security
 Sorting
 Text formatting
 Transforming
+
+Programming Style
+---
+Behavior driven development
+Functional programming
+Object oriented programming
+Reactive programming
+Test driven development
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Test-driven_development), it is written with a dash: Test-driven development